rust_src/src/chartable.rs

```diff
@@ -166,41 +175,48 @@ impl LispCharTableRef {
         // char table is 4 LispObjects + an array
         size1 -= 4;
 
-        unsafe {
-            if !internal_equal(self.defalt, other.defalt, kind, depth + 1, ht) {
-                return false;
-            }
-            if !internal_equal(self.parent, other.parent, kind, depth + 1, ht) {
-                return false;
-            }
-            if !internal_equal(self.purpose, other.purpose, kind, depth + 1, ht) {
-                return false;
-            }
-            if !internal_equal(self.ascii, other.ascii, kind, depth + 1, ht) {
-                return false;
-            }
+        if !self
+            .defalt
```

Comment:
That's can't be right.

Reply:
Yeah, it is a typo in the C source.

Reply:
I think maybe because of a C compiler declaring 'default' a reserved work for case statements?

Reply:
Yuck.